README.md

```diff
@@ -36,19 +40,26 @@ nanoff --target M5StickCPlus --update --preview --serialport COM3 --baud 115200
 # For the M5Core2:
 nanoff --target M5Core2 --update --preview --serialport COM3
 ```
+NOTE3: If the nanoff commands fail, make sure you did what is written on NOTE1.
 
 Once you have the nugets, you can then enjoy accessing the screen, the accelerometer, get a Grove I2C connecter, add events on the buttons. And you don't even need to think about anything, all is done for you in the most transparent way!
 
 > Note: All the classes that you'll have access are all using the Lazy pattern to be instantiated including the screen. This have the advantage to use as little memory and setup time as possible.
 
 In the samples below, we'll use either M5Stack, either M5Stick as examples, they are all working in a very similar way.
 
+### namespaces
+
+Make sure you add the proper namespaces reference to your C# program header e.g.
+using nanoFramework;
+
 ### Screen
 
-The only thing you need to do to access the screen is to initialize it:
+The only thing you need to do to access the screen is to initialize it; nevetheless this is also target specific e.g.:
 
 ```csharp
-M5Stack.InitializeScreen();
+For Core use: M5Stack.InitializeScreen();
```

Comment:
These code blocks are meant to be copy/paste, so the code inside has to be valid.
If you want to add examples for Core and StickCPlus you have to have to 2 code blocks with the note before each one explaining on which target it belongs to.

Reply:
ok , i understand...but i would suggest that you can implement that small change as you did with the text ,...otherwise the overhead is too high...and i dont actually understand yet how should i proceed now...on Github i have two options, either "Commit suggestion" or "Resolve Conversation"...what does the later do??

Reply:
@jcabad100 check [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#resolving-conversations). 
You should mark a conversation as resolved if you're addressing directly the comment with a change that matches the suggestion of the reviewer. If you replying back or adding new ideas/suggestions, you should leave it open. Then it could be up to you or the reviewer to close it.

As for the overhead you mention, not sure exactly what you mean... if this about the changes, know that you can make small changes directly on github or on an open PR.